scripts/manage_api.py

```diff
@@ -72,34 +72,39 @@ def main(self):
             id_info = 'automated test key'
 
         LOGGER.info('making key for {0}:{1}'.format(username, id_info))
-
-        api_db = TinyDB(self.cache_path)
-
-        current_key = api_db.search(
-            Query().user_name == username
-        )
+        
+        #Connect to TinyMongoDB and use prosperAPI DB 
+        connection = TinyMongoClient(CACHE_PATH)
+        api_db = connection.prosperAPI
+        #Attach to users collection
+        usersDB = api_db.users
+
+        current_key = usersDB.find_one({'user_name': username })
+        
         if current_key:
+        
             key_msg = \
             'user already has a key' + \
-            '\n\tapi_key={0}'.format(current_key[0]['api_key']) + \
-            '\n\tuser_name={0}'.format(current_key[0]['user_name']) + \
-            '\n\tuser_info={0}'.format(current_key[0]['user_info']) + \
-            '\n\tkey_generated={0}'.format(current_key[0]['key_generated']) + \
-            '\n\tlast_accessed={0}'.format(current_key[0]['last_accessed'])
-
+            '\n\tapi_key={0}'.format(current_key['api_key']) + \
+            '\n\tuser_name={0}'.format(current_key['user_name']) + \
+            '\n\tuser_info={0}'.format(current_key['user_info']) + \
+            '\n\tkey_generated={0}'.format(current_key['key_generated']) + \
+            '\n\tlast_accessed={0}'.format(current_key['last_accessed'])
+            
             print(key_msg)
             if not self.debug:
                 LOGGER.info(key_msg)
 
             if not self.force:
                 exit()
-
+        """
```

Comment:
So that is part of code block  `        if current_key and not self.debug:
            api_db.remove(
                Query().user_name == username
            )` I am not sure what that does and commented it out for the time being. Is this still used? If so I need to rework it into mongo syntax